project/AkkaBuild.scala

```diff
@@ -213,103 +202,34 @@ object AkkaBuild extends Build {
     dependencies = Seq(persistence % "compile;provided->provided;test->test", testkit % "compile;test->test")
   )
 
-  lazy val streamAndHttp = Project(
```

Comment:
please keep this project, I use it a lot. (to test both streams and http with one test command)